data/overview.yaml

```diff
@@ -13,9 +13,9 @@ group:
     orbit: 'group_theory/group_action.html#mul_action.orbit'
     quotient space: 'group_theory/group_action.html#mul_action.orbit_equiv_quotient_stabilizer'
     class formula: ''
-    conjugacy classes: ''
+    conjugacy classes: 'algebra/group/conj.html'
```

Comment:
I'm not sure if this is really conjugacy *classes*. But we have all the ingredients.

Reply:
I don't think we should cheat in this file. Leaving this line empty creates an incentive to actually fix this tiny gap. We can even hope that the person fixing this gap will use the momentum to add some more.